include/drivers/interrupt_controller/gic.h

```diff
@@ -196,4 +196,24 @@
 
 #endif /* CONFIG_GIC_VER <= 2 */
 
+#ifndef _ASMLANGUAGE
+
+/*
+ * GIC Driver Interface Functions
+ */
+
+int arm_gic_init(void);
```

Comment:
It would be nice to add some documentation here, for these functions as they are generic to ARM

Reply:
Added docs for all `arm_gic_` functions